spec/features/admin/product_import_spec.rb

```diff
@@ -33,7 +33,7 @@
 
     it "validates entries and saves them if they are all valid and allows viewing new items in Bulk Products" do
       csv_data = CSV.generate do |csv|
-        csv << ["name", "supplier", "category", "on_hand", "price", "units", "unit_type"]
+        csv << ["name", "enterprise", "category", "on_hand", "price", "units", "unit_type"]
```

Comment:
I think it's good practice to have code and spec changes in the same commit. It makes the use of git bisect possible and shows the impact of a code change.